### PR TITLE
Refactoring of `Schema::normalizeDefaultValue()` method

### PR DESCRIPTION
**Pull request by Tigrov**
* `CURRENT_TIMESTAMP` for default value was not `Expression()`
* fix empty strings '' in default value
* fix single quote in default value
* fix `DEFAULT NULL` definition

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by vjik — COMMENTED** [review]
Please, add line to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Enh #226: Add support for auto increment in primary key column. (@terabytesoftw)
 - Enh #225: Typecast refactoring (@Tigrov)
+- Bug #229: Refactoring of `Schema::normalizeDefaultValue()` method (@Tigrov)
```
Comment (vjik) [inline CHANGELOG.md:7]: Seems, here not only refactoring, but bugfixes:

- CURRENT_TIMESTAMP for default value was not Expression()
- fix empty strings '' in default value
- fix single quote in default value
- fix DEFAULT NULL definition

?

Comment (Tigrov) [reply CHANGELOG.md]: Yes, rigth. Thanks

**Review by vjik — APPROVED** [review]

**Review by terabytesoftw — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**